---
name: Bug report
about: Create a report to help us improve Awesome-SSO's quality
title: ""
labels: "T: bug"
assignees: ""
---

**Describe the bug**

<!-- A clear and concise description of what the bug is. -->

**To Reproduce**

<!--
Minimal steps to reproduce the behavior with source code and configuration.
-->

For example, take this code:

```python
this = "code"
```

And run it with these arguments:

```sh
$ pytest
```

The resulting error is:

> cannot format file.py: INTERNAL ERROR: ...

**Expected behavior**

<!-- A clear and concise description of what you expected to happen. -->

**Environment**

<!-- Please complete the following information: -->

- awesome-sso version: <!-- e.g. [main] -->
- OS and Python version: <!-- e.g. [Linux/Python 3.7.4rc1] -->

**Additional context**

<!-- Add any other context about the problem here. -->
